runner/src/artifacts.rs

```diff
@@ -211,7 +227,7 @@ fn objcopy(target: &Target) -> PathBuf {
     let mut bin_path = path.clone();
 
     match target {
-        Target::Miralis => {
+        Target::Miralis | Target::MiralisAsFirmware => {
             elf_path.push("miralis");
             bin_path.push("miralis.img");
```

Comment:
Doesn't `miralis.img` get overwritten when we build Miralis twice? In the end we need two images, one as the host Miralis and one as the virtualized Miralis, how do we ensure we keep both?

Reply:
Looking at the code below it seems they use two different targets, so they end up in two different directories which solves the issue :)

Reply:
Exactly, we don't have any issues even if the name is the same